votor/src/event_handler/stats.rs

```diff
@@ -0,0 +1,363 @@
+use {
+    crate::{event::VotorEvent, voting_utils::BLSOp, VoteType},
+    solana_metrics::datapoint_info,
+    solana_votor_messages::bls_message::BLSMessage,
+    std::time::{Duration, Instant},
+};
+
+const STATS_REPORT_INTERVAL: Duration = Duration::from_secs(10);
+
+#[derive(Debug)]
+pub(crate) struct EventHandlerStats {
+    pub(crate) commitment_updates: u16,
+    pub(crate) ignored: u16,
+    pub(crate) leader_window_replaced: u16,
+    pub(crate) set_root: u16,
+    pub(crate) timeout_set: u16,
+
+    /// All of the following fields are in microseconds
+    pub(crate) receive_event_time: u32,
+    pub(crate) send_vote_time: u32,
+
+    pub(crate) received_events_count_and_timing: Vec<(u16, u32)>,
+    pub(crate) sent_votes: Vec<u16>,
```

Comment:
Might be easier to understand/maintain with just `HashMap` types mapping from the eventtypes/vote types to the counts, easier to manage and performance not really a concern here